frontend/src/core/components/DefaultLayout/DefaultLayout.js

```diff
@@ -4,14 +4,51 @@ import {Redirect, Route} from "react-router-dom";
 import AppSidebar from "./AppSidebar/AppSidebar";
 import AppNavbar from "./AppNavbar/AppNavbar";
 import Breadcrumbs from "./BreadCrumb/Breadcrumb";
-import {dashboardRoutes, ROUTE_PATHS} from "../../../_routes";
+import {dashboardRoutes, ROUTE_PATHS, BREADCRUMBS} from "../../../_routes";
 import UserService from "../../services/PocketUserService";
 import "./DefaultLayout.scss";
+import isArray from "lodash/isArray";
 
 class DefaultLayout extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.onBreadCrumbChange = this.onBreadCrumbChange.bind(this);
+
+    this.breadcrumbss = [];
+
+    this.state = {
+      breadcrumbs: [],
+    };
+  }
+
+  onBreadCrumbChange(breadcrumbs) {
+    this.setState({breadcrumbs});
+  }
+
   render() {
     // eslint-disable-next-line react/prop-types
     const {path} = this.props.match;
+    let {breadcrumbs} = this.state;
+    let breadcrumbsShow = breadcrumbs;
+
+    // eslint-disable-next-line react/prop-types
+    const route_path = this.props.history.location.pathname.replace(
+      ROUTE_PATHS.home, "");
+
+    const breadcrumbsMapFn = (br, idx, arr) => {
+      return {label: br, active: idx === arr.length - 1};
+    };
+
+    if (route_path in BREADCRUMBS()) {
```

Comment:
IMO maybe you should change BREADCRUMBS to be lower case and maybe a different name?